code/src/details.css

```diff
@@ -0,0 +1,175 @@
+.detailsWrapper {
+    position: relative;
+}
+
+.detailsWrapper .backBtn {
+    position: absolute;
+    display: inline-flex;
+    color:#FFF;
+    background-color: transparent;
+    border-style: none;
+    align-items: center;
+    font-weight: bold;
+    text-decoration: none;
+    left: 50px;
+    top: 50px;
+    z-index: 1;
+    font-size: 18px;
+}
+
+.btnWrapper .backBtn {
+    position: absolute;
+    top: 0;
+    margin-top: 5%;
+}
+
+.backBtn {
+    position: absolute;
+}
+
+.backBtnName {
+    margin-left: 5px;
+}
+
+.backBtnName:hover {
+    margin-left: 10px;
+}
+
+.background {
```

Comment:
When I look at your site on desktop, there is a black bar down the right-hand side where the background image doesn't stretch the full width of the screen.  I wonder if this is because it is a regular image/div styled as a background and not a background image?  Or is there something you could change in the CSS to get the background to cover the full width of larger screens?